umbraco-cloud/set-up/project-settings/availability-performance.md

```diff
@@ -0,0 +1,106 @@
+
+### Panel view
+
+The panel selector consists of four tiles, each representing a specific segment of data. The four segments are failed request, App Performance, CPU Usage, and Memory Usage.
+
+![Pane-view](../images/AP-Panel-Selector.png)
+
+Each tile includes relevant statistics and potentially a warning or an error indicator in case there is something you might want to consider.
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
+- Failed Requests: when one or more client errors (but no server errors) have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
+
+Errors and warnings for CPU Usage and Memory Usage are only shown for cloud projects on a shared plan and with a granularity of 5 minutes selected.
+
+### Chart and statistics view
+
+For each segment, there will be shown a chart and a set of related statistics.
+
+#### Failed request
+The chart shows the breakdown of HTTP status codes for each data point with the selected granularity. Only responses indicating a client (4xx region) or server errors (5xx region) are shown.
+
+![Failed-request](../images/AP-1-FailedRequests.png)
+
+In the statistics panel on the right, you will find the total instances of the status code in the time range.
+
+#### App Performance
+The chart shows the average response time during the selected time range. All requests to the Umbraco solution in the time periods with the length of the selected granularity count to average response time.
+
+![AppPerformance](../images/AP-2-AppPerformance.png)
+
+The statistics panel shows the average, maximum, and minimum response for the shown data points.
+
+#### CPU Usage
+The chart depicts the CPU time consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Cpu usage](../images/AP-3-CpuUsage2.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned CPU time in seconds and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>. 
+In this case, the statistics panel maximum CPU time, average CPU, plan quota, and the maximum and average percentage of the consumed CPU in a 5 minute period compared to the plan quota.
```

Comment:
```suggestion
In this case, the statistics panel shows the following:
- The maximum CPU time 
- The average CPU
- The plan quota
- The maximum and average percentage of the consumed CPU in a 5-minute period compared to the plan quota.
```

Reply:
Yup, nice improvement. The second line "The average CPU" should include " time" as well though.

Reply:
great I added it to the suggestion :)